winml/test/scenario/cppwinrt/CustomOps.cpp

```diff
@@ -724,7 +717,19 @@ TEST_F(CustomOpsScenarioTest, CustomKernelWithCustomSchema)
         {
             // Check that the shape inference context is closed and safely fails
             MLOperatorEdgeDescription edgeDesc;
-            EXPECT_EQ(E_INVALIDARG, shapeInferenceContext->GetInputEdgeDescription(0, &edgeDesc));
+            WINML_EXPECT_EQUAL(E_INVALIDARG, shapeInferenceContext->GetInputEdgeDescription(0, &edgeDesc));
         }
     }
 }
+
+const CustomOpsTestApi& getapi() {
```

Comment:
The whole point of defining this getapi layer of indirection is to be able to move all the custom runtime filtering of test behavior into one place. So need to make sure to add this into every getapi() implementation, and then remove the old macros we were using like GPUTEST and EDGECORE.

Reply:
I'll create a task to track this